docker/elo_db/db_schema.sql

```diff
@@ -32,12 +32,12 @@ USE elo_app;
 --
 
 CREATE TABLE `games` (
-  `game_nid` int(11) NOT NULL,
```

Comment:
Miałem silne podejrzenia, że phpMyAdmin nie lubi się z takimi kodami

Reply:
hmm, powinien chyba to łykać, dziwne

Reply:
Wyeksportowałem tą tabelkę z phpmyadmina i wygląda tak:
``
CREATE TABLE `games` (
  `game_nid` int(11) NOT NULL,
  `winner_user_nid` int(11) NOT NULL,
  `looser_user_nid` int(11) NOT NULL,
  `winner_rating_before` int(11) NOT NULL,
  `looser_rating_before` int(11) NOT NULL,
  `rating_diff` int(11) NOT NULL,
  `cdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
``

Reply:
Według mnie to powinno zostać

Reply:
@ZielonyBuszmen 
Też sie zastanawiam czemu tutaj masz jakieś podejrzenia :>. Czemu się nie lubi?

Reply:
@ktrzos Brzytwa Ockhama ucięła tutaj za dużo, a winną była tylko jedna linijka (linia 78, `  ADD UNIQUE KEY `code` (`code`(10));`)